portfolio/src/main/java/com/google/sps/data/MapMarker.java

```diff
@@ -0,0 +1,37 @@
+// Copyright 2019 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package com.google.sps.data;
+
+import java.io.IOException;
+import java.util.ArrayList;
+import java.util.List;
+
+public class MapMarker {
+    private int lat;
+    private int lng;
```

Comment:
I believe you need to use `double` types for both lat and long. That is how maps api would take it.

Reply:
yes indeed! I had it at int for testing purposes but I should update it now :)